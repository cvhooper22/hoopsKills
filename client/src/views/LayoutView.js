import React, { Fragment } from 'react';
import PropTypes from 'prop-types';

function LayoutView (props) {
  const { renderPageContent } = props;

  return (
    <Fragment>
      <div className="container">
        <div className="row">
          <div className="col-md-12">
            { renderPageContent() }
          </div>
        </div>
      </div>
    </Fragment>
  );
}

LayoutView.propTypes = {
  renderPageContent: PropTypes.func,
};

export default LayoutView;
