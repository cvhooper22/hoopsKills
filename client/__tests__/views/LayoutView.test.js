import React from 'react';
import renderer from 'react-test-renderer';
import LayoutView from '../../src/views/LayoutView';

describe('LayoutView -', () => {

  function renderPageContent () {
    return (
      <div>page content</div>
    );
  }

  it('Should render LayoutView', () => {
    const view = renderer.create(
      <LayoutView
        renderPageContent={ renderPageContent }
      ></LayoutView>
    );

    expect(view).toMatchSnapshot();
  });

});