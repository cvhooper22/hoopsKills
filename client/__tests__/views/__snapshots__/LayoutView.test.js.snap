// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutView - Should render LayoutView 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <div>
        page content
      </div>
    </div>
  </div>
</div>
`;
